zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/CamundaExporter.java

```diff
@@ -213,6 +227,24 @@ private void flushAndReschedule() {
     scheduleDelayedFlush();
   }
 
+  private void scheduleImportersCompletedCheck() {
+    controller.scheduleCancellableTask(
+        Duration.ofSeconds(10), this::checkImportersCompletedAndReschedule);
```

Comment:
🔧 Do we want to make this interval configurable?

Reply:
I didn't think there was an issue with making it configurable, but to avoid any possible issues with this field it's probably better to be not configurable

Reply:
I don't think there's any reason a user would want to configure this.